### DISCUSSION TITLE
<help>

### DISCUSSION THREAD

**Discussion by raniagomaa123456** [community/community#68285] [Discussions]
### Select Topic Area

Question

### Body

I have already been approved for the jet Hub student package, so whenever I try to book a Mongo certificate, this message appears to me

Welcome and thanks for logging in. Unfortunately we are unable to verify your status as a participant in the GitHub Student Developer Pack. Please visit the GitHub Student Pack home for additional information and how to register.

Approved Submitted Aug 31, 2023
Congrats! Your request for [@raniagomaa123456](https://github.com/raniagomaa123456) was approved and your benefits are now available.

**Comment by queenofcorgis** [comment]
Hi @raniagomaa123456 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post. As a reminder,, the Discussions category is solely related to conversations around the GitHub product Discussions. 

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
